pkg/jiraservicedesk/client/client_test.go

```diff
@@ -0,0 +1,115 @@
+package client
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/nbio/st"
+	"gopkg.in/h2non/gock.v1"
+)
+
+const BaseURL = "https://sample.atlassian.net/"
+
+func TestJiraService_GetProject_shouldGetProjectById(t *testing.T) {
+	defer gock.Off()
+
+	// The project.LeadAccountId & project.ProjectTemplateKey are not returned by the GET call. Therefore, those values are not returned and matched via Gock Server
+	gock.New(BaseURL + "rest/api/3/project").
```

Comment:
For slug(the part after base url) use https://github.com/stakater/jira-service-desk-operator/blob/master/pkg/jiraservicedesk/client/client_project.go#L14 instead of hard-coding it here.

In that case we'll just have a single place to update the path if it's required.